app.server.js

```diff
@@ -35,38 +36,66 @@ const startServer = async () => {
 
   app.post("/continue/:originator", (req, res) => {
     const currentPage = `/${req.params.originator}`;
+    const sessionData = req.session || {};
     const formData = req.body;
-    let sessionData = {};
-    if (formData.sessionData) {
-      sessionData = JSON.parse(formData.sessionData);
-    }
-    const previousAnswers = sessionData.answers || {};
 
+    const previousAnswers = sessionData.cumulativeAnswers || {};
     let newAnswers = Object.assign({}, formData);
-    delete newAnswers["sessionData"];
 
     const cumulativeAnswers = Object.assign(previousAnswers, newAnswers);
```

Comment:
This feels like a little bit too much logic inside an server.js file - not sure on next.js best practice, do they have concepts of controllers? Somewhere between simple "validation" (which normally does live in the routing) and the "business logic" of editing data. Appreciate it's a little strange at the moment because we're not doing much. It also feels like there should be a "routing" file outside of the server.js. Let's discuss